### PR TITLE
Added some more information to avoid issues while working on PR

### PR DESCRIPTION
**Pull request by mish24**
1. Added the info that all new PRs must originate from the `master` branch.
2. While creating a new IA page, it must be ensured that the name ends with `_cheat_sheet` so that the IA page and the id in the json file are same, and it's triggered.

### CODE REVIEW

File: walkthroughs/programming-syntax.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ If this is your first time developing an Instant Answer, check out our [detailed
 
 ## Create the Instant Answer Page
 
-> If you're just using this walkthrough to learn and practice for now, you can skip this step. This is important only if you plan to submit what you're working on.
+> If you're just using this walkthrough to learn and practice for now, you can skip this step. This is important only if you plan to submit what you're working on. 
 
 Every Instant Answer on DuckDuckGo.com has its very own [*Instant Answer Page*](https://duck.co/ia/). These are the home base for planning, collaboration, and metadata. Instant Answer pages also show any Github issues and let you know what stage the Instant Answer is in.
 
-- If you're building a brand new cheat sheet, start by [creating a new Instant Answer page](https://duck.co/ia/new_ia).
+- If you're building a brand new cheat sheet, start by [creating a new Instant Answer page](https://duck.co/ia/new_ia). Please remember to end the name of the Instant Answer with `_cheat_sheet` so that the id in the json file and the page are same.
```
Comment (J-tt) [inline walkthroughs/programming-syntax.md]: Should this also mention capitalization rules?

File: walkthroughs/calculation.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Next, change into the Goodie repository's home directory, `zeroclickinfo-goodies
 [01:07 PM codio@border-carlo workspace ]$ cd zeroclickinfo-goodies
 ```
 
-Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time.
+Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time. Also, make sure that you make the new branches from the master branch only.
```
Comment (J-tt) [inline walkthroughs/calculation.md]: This looks all good, feels quite wordy. Perhaps 'Make sure that the branch is a fork of the DuckDuckGo master, and not any other branch' as it avoids duplicating words, while also being specific on what master branch to use.

**Review by J-tt — CHANGES_REQUESTED** [review]
This looks really good, just needs a few touches and it'll be great!

File: walkthroughs/programming-syntax.md (hunk 1 above)
Comment (mish24) [reply walkthroughs/programming-syntax.md]: Yeah! I'll add that too

**Comment by mish24** [comment]
@J-tt Hi, I added the proposed language changes. Please review :)

File: walkthroughs/calculation.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Next, change into the Goodie repository's home directory, `zeroclickinfo-goodies
 [01:07 PM codio@border-carlo workspace ]$ cd zeroclickinfo-goodies
 ```
 
-Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time. Also, make sure that you make the new branches from the master branch only.
+Before doing any coding, it's recommended you work from a [new, separate Git branch](http://docs.duckduckhack.com/resources/git-workflow.html#step-3-create-a-working-branch-for-coding) *and not master*. A separate branch allows you to keep your repository up-to-date, and work on multiple pull requests at one time. Also, make sure that the branch is a fork of the DuckDuckGo master.
```
Comment (J-tt) [inline walkthroughs/calculation.md]: Definitely a lot more clear! Good work 👍

**Review by J-tt — APPROVED** [review]
Looks good!

**Review by moollaza — APPROVED** [review]

**Comment by moollaza** [comment]
@mish24 thanks for the PR. I've made some edits.

**Pull request merged**